### PR TITLE
Package horned_worm.0.3.6

### PR DESCRIPTION
**Pull request by kkazuo**
### `horned_worm.0.3.6`
An easy functional Web app micro framework
An easy functional Web app micro framework.
You can create REST API server with composable functions.

---
* Homepage: https://github.com/kkazuo/horned_worm
* Source repo: git+https://github.com/kkazuo/horned_worm.git
* Bug tracker: https://github.com/kkazuo/horned_worm/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: packages/horned_worm/horned_worm.0.3.6/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+bug-reports: "https://github.com/kkazuo/horned_worm/issues"
+dev-repo: "git+https://github.com/kkazuo/horned_worm.git"
+depends: [
+  "ocaml"           {>= "4.03.0"}
+  "async"           {>= "v0.11.0" & < "v0.13.0"}
+  "cohttp-async"    {>= "1.0.2" & < "3.0.0"}
+  "logs"            {>= "0.6.2"}
+  "re"              {>= "1.7.3"}
+  "yojson"          {>= "1.6.0"}
+  "dune"            {build & >= "1.0.1"}
```
Comment (mseri) [inline packages/horned_worm/horned_worm.0.3.6/opam:20]:
```suggestion
  "dune"            {>= "1.0.1"}
```

Dune cannot be tagged as build dependency due to forward compatibility issues (see https://github.com/ocaml/opam-repository/issues/14185)

Comment (mseri) [reply packages/horned_worm/horned_worm.0.3.6/opam]: Can you upstream the change?

**Comment by kkazuo** [comment]
Thank you @mseri .
I publish fixed .opam file at #17579 .

**Comment by mseri** [comment]
Thanks, there was no need to resubmit, fixing metadata here was enough, but that is fine as well :)

**Pull request closed without merging**